include-in-header.html

```diff
@@ -107,18 +107,25 @@
         });
     }
 
-    function addPanelPillsNav() {
+    function handlePanelVariants() {
```

Comment:
`.panel-pills` and `.panel-underline` should be core Quarto features! I think you should consider PR-ing this to quarto or at least putting this function in an issue so others can use it.

That said, you should leave a note here linking to that issue or at least a reminder that we'd want to take this out if `.panel-pills` ships in quarto.

(I could see quarto using `.panel-tabset-underline` and `.panel-tabset-pills` too, similar to their `.callout-{tip,warning,...}` classes.)